src/pybind/mgr/cephadm/template.py

```diff
@@ -29,7 +29,7 @@ class Jinja2Engine(TemplateEngine):
     def __init__(self):
         self.env = Environment(
             loader=PackageLoader('cephadm', 'templates'),
-            autoescape=select_autoescape(['html', 'xml']),
+            autoescape=select_autoescape(['html', 'xml'], default_for_string=False),
```

Comment:
just out of curiosity, why do we need `select_autoescape(['html', 'xml']`. cc @bk201  ?

Reply:
The code was taken from the official document, it's recommended for security reason for rendering web pages. I don't think we need it.